2-api/limits.md

```diff
@@ -2,6 +2,10 @@
 
 # Limits and errors
 
+The Perspective API uses limits and quotas to protect the system from receiving more data than it can handle, and to ensure equitable distribution of the system resources. These limits and quotas are subject to change.
+
+You may encounter an error if your request exceeds the limit.
+
 ## Quota limit
 
 By default, we set the quota to 1 QPS for all Perspective projects.
```

Comment:
Would it be too much to specific that QPS means "query per second" here? It's a commonly used acronym, but I think it's good to clarify acronyms the first time they're used, in case the user hasn't seen them before.

Reply:
Absolutely!